sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -341,12 +341,25 @@ export class ChatClient {
    */
   public on(event: "participantsRemoved", listener: (e: ParticipantsRemovedEvent) => void): void;
 
-  public on(event: ChatEventId, listener: (e: any) => void): void {
+  /**
+   * Subscribe function for realTimeNotificationConnected.
+   * @param event - The realTimeNotificationConnected Event
+   * @param listener - The listener to handle the event.
+   */
+  public on(event: "realTimeNotificationConnected", listener: (e: ConnectionState) => void): void;
```

Comment:
We don't need a paramter (e: ConnectionState) for listener if we expose 2 events.

Reply:
removed it